package.json

```
@@ -1,9 +1,9 @@
 {
   "scripts": {
-   "dev": "npx tailwindcss -i ./src/input.css -o ./dist/output.css --watch",
-   "build": "npx tailwindcss -i ./src/input.css -o ./dist/output.css"
+    "dev": "npx tailwindcss -i ./src/input.css -o ./dist/output.css --watch",
+    "build": "npx tailwindcss -i ./src/input.css -o ./dist/output.css"
```

It seems like there's additional spaces on this script commands, you may remove them
```suggestion
   "dev": "npx tailwindcss -i ./src/input.css -o ./dist/output.css --watch",
   "build": "npx tailwindcss -i ./src/input.css -o ./dist/output.css"
```